source/reference/compatibility.txt

```diff
@@ -205,27 +205,31 @@ in ``mongosh`` better align with the types used by the MongoDB Drivers.
 Undefined Values
 ----------------
 
-The undefined BSON type is `deprecated <https://bsonspec.org/spec.html>`__ in
-:binary:`bin.mongosh`. If you insert a document with the undefined JS type in
-:binary:`bin.mongosh`, your deployment replaces the undefined value in your 
+The undefined BSON type is `deprecated <https://bsonspec.org/spec.html>`__. 
+If you insert a document with the undefined JS type in
```

Comment:
[question]

Should this be "undefined BSON type" on line 209?

Reply:
Thanks for taking a look and noticing this, @jmd-mongo ! I think in [a past PR](https://github.com/mongodb/docs-mongodb-shell/pull/364/files/f0d216ca134362965ba6ece21b9220262944f4bc#r1829635518) that added this "Undefined Values" section to this page, @addaleax noted that line 209 should say "undefined JS type" instead of "undefined BSON type", since you can't create an undefined BSON type in mongosh. 

But I agree it is a bit confusing the way it's written currently! I reworded it so that it says "If you try to insert a document with [...]". Lmk what you think!

Reply:
Thanks for the clarification!